BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/location/Location.java

```diff
@@ -113,6 +113,10 @@ public boolean latitudeAndLongitudeEquals(Location location) {
         return latitude == location.latitude && longitude == location.longitude;
     }
 
+    public boolean latitudeAndLongitudeEqualsWithDelta(Location location, double delta) {
+        return (latitude - location.latitude < delta) && (longitude - location.longitude < delta);
```

Comment:
This is wrong, the calculated delta needs to be absolute. Imagine `this.latitude = 1000` and `location.latitude = 2000`. Also, you can overload the `latitudeAndLongitudeEquals(...)` method, as the signature is different.